src/styles/_colors.css

```diff
@@ -4,4 +4,5 @@
   --color-text: #d84064;
   --color-items-background: #f4cbad;
   --color-single-card: #c05757;
+  --color-button-text: whitesmoke;
```

Comment:
You can convert this one into hex too so you have consistency in all your colors.